MVP.md

```diff
@@ -1,27 +1,7 @@
 # Minimum Viable Product
 
-As stated in the [high-level goals](HighLevelGoals.md), the first release aims
-at being a Minimum Viable Product (MVP). This means that there are important
-features we *know* we want and need, but are post-MVP; these are in a separate
-essential [post-MVP :unicorn:][future general] features document. The MVP will contain
-features which are available today in modern web browsers and which perform well
-even on mobile devices, which leads to roughly the same functionality as
-[asm.js](http://asmjs.org).
+WebAssembly initially launched with a "minimum viable product" version (the "MVP"),
+which was complete enough for some use cases, however many important features were
+deferred. WebAssembly has since add many [features].
```

Comment:
I'm not a native English speaker, so I'm not 100% sure but let me suggest a correction:

```suggestion
deferred. WebAssembly has added many [features] since then.
```

Reply:
Good spot, that is indeed an error. I had intended "has since added many features". Your correction is also correct, so I've committed it.

Reply:
Thank you. I didn't know the usage of "since" as an adverb.